concepticondata/references/references.bib

```diff
@@ -1,4 +1,4 @@
-@dataset{Sidwell2021,
+@book{Sidwell2021,
```

Comment:
Sorry, the Bibtex entry should be like:

```
Sidwell, Paul (2021): Nicobarese 100 item wordlist for phylogenetic analyses [Data Set, Version 1.0]. Original Talk: Classification of the Nicobarese languages. Presented at the 9th International Conference on Austroasiatic Linguistics (ICAAL). Lund: Centre for Languages and Literature at Lund University. DOI: https://doi.org/10.5281/zenodo.5508350.
```
As Bibtex this would be:

```
@book{Sidwell2021,
  author = {Sidwell, Paul},
  year = {2021},
  title = {Nicobarese 100 item wordlist for phylogenetic analyses [Data Set, Version 1.0]},
  howpublished = {Original Talk: Classification of the Nicobarese languages. Presented at the 9th International Conference on Austroasiatic Linguistics (ICAAL)},
  address = {Lund},
  publisher = {Centre for Languages and Literature at Lund University},
  doi = {https://doi.org/10.5281/zenodo.5508350}
}
```

Reply:
Can you update this also in the lexibank repostiory?